gqcp/include/Basis/TransformationMatrix.hpp

```diff
@@ -56,6 +58,51 @@ class TransformationMatrix : public SquareMatrix<_TransformationScalar> {
 
         (*this) = (*this) * T;
     }
+
+    /**
+     *  @return returns this transformation as to the $VEC group format of GAMESS-US
+     */ 
+    std::string asGamessUsVecGroup() {
```

Comment:
I'd keep the method name in line with the actual name of the problem, similar to FCIDUMP.

```suggestion
    std::string asGAMESSUSVECGroup() const {
```